### DISCUSSION TITLE
How to allow GPU?

### DISCUSSION THREAD

**Discussion by chrisborell** [nextcloud/all-in-one#4844] [Questions] [answered]
Hi,

I'm currently using Unraid with Nextcloud AIO in Docker. How can I passthrough the GPU to Nextcloud for features like Memories and Recognize, and future AI functionalities?

Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/4277
